teamcity/datadog_checks/teamcity/data/conf.yaml.example

```
@@ -45,44 +13,412 @@ init_config:
 #
 instances:
 
-    ## @param name - string - required
-    ## A custom unique name per build configuration that is shown
-    ## in the events.
+    ## @param server - string - required
+    ## Specify the server name of your TeamCity instance.
+    ## Enable Guest Authentication on your instance or specify `username` and `password` to 
+    ## enable basic HTTP authentication.
     #
-  - name: <BUILD_NAME>
+  - server: http://teamcity.<ACCOUNT_NAME>.com
 
-    ## @param server - string - required
-    ## Specify the server name of your teamcity instance
-    ## Guest authentication must be on if you want the check to be able to get data
-    ## When using the optional `basic_http_authentication` use:
+    ## @param username - string - optional
+    ## The username to use if services are behind basic or digest auth.
+    #
+    # username: <USERNAME>
+
+    ## @param password - string - optional
+    ## The password to use if services are behind basic or NTLM auth.
+    #
+    # password: <PASSWORD>
+
+    ## @param use_openmetrics - boolean - optional - default: false
+    ## Use the latest OpenMetrics V2 implementation to collect metrics from
+    ## the TeamCity server's Prometheus metrics endpoint.
+    ## Requires Python version 3.
+    ##
+    ## Enable in a separate instance to collect Prometheus metrics.
+    ## This option does not collect events, service checks, or metrics from the TeamCity REST API.
```

```suggestion
    ## This option does not collect events or service checks from the TeamCity REST API.
```
There are no metrics in the REST API are they?

---

Yes, there are metrics collected from the REST API